client/src/databrowser/test/databrowser.unit.spec.js

```diff
@@ -47,8 +41,9 @@ tape('levels before variable, type, and categories - no gaps', function(test) {
 			id: 'A1a',
 			name: 'A.1.a',
 			type: 'categorical',
-			values: { '0': { label: 'No' }, '1': { label: 'Yes' } },
+			values: { 0: { label: 'No' }, 1: { label: 'Yes' } },
 			groupsetting: { inuse: false },
+			additionalAttributes: {},
```

Comment:
@xzhou82 This parameter was added to parseDictionary() but wasn't documented. Would you please tell me what this is so I can add it to the SOP?